logic/service/raw-data-service.js

```diff
@@ -633,8 +699,12 @@ exports.RawDataService = DataService.specialize(/** @lends RawDataService.protot
      */
     mapRawDataToObject: {
         value: function (record, object, context) {
+            var blueprint = this._bluePrintForObject(object);
```

Comment:
if there's this.mapping then blueprint wouldn't be used, so let's try to do initialize it when we know we have to?